src/main/java/org/jenkinsci/plugins/mesos/JenkinsScheduler.java

```diff
@@ -82,15 +94,15 @@
 
     private static final Logger LOGGER = Logger.getLogger(JenkinsScheduler.class.getName());
 
-    public static final Lock SUPERVISOR_LOCK = new ReentrantLock();
+    public static final ReentrantLock SUPERVISOR_LOCK = new ReentrantLock();
```

Comment:
Is this change necessary? Seems like we don't need to go from an interface to a more concrete version in the definition

Reply:
It's for clearance, indicating this lock is preventing reentrance only.